marlon/deepq/simulate.py

```diff
@@ -36,19 +37,17 @@ def run_simulation():
     dql_run = torch.load('deepq.pkl')
     learner = dql_run['learner']
 
-    iteration_count = 1000
+    iteration_number = 1000
+    simulation = [generate_graph_json(gym_env, 0)]
 
-    simulation = [generate_graph_json(gym_env)]
-    for _ in range(iteration_count - 1):
+    for iteration_count in range(iteration_number):
```

Comment:
Variables names should be switched; `iteration_count` being the max and `iteration_number` being the current iteration.